src/App/components/Header/components/AddLocation/components/DrawerBlock/DrawerBlock.module.scss

```diff
@@ -115,6 +115,22 @@
       font-family: roboto-medium;
     }
   }
+
+  .foundLocationDetailsContainer {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .flagContainer {
+    margin-right: 10px;
+    height: 15px;
+    width: 20px;
```

Comment:
Let's use here variables that we already have:
$space-xs: 10px;
$space-s: 15px;
$space-m: 20px;